rudder-core/src/main/scala/com/normation/rudder/services/marshalling/XmlUnserialisationImpl.scala

```diff
@@ -161,7 +161,7 @@ class DirectiveUnserialisationImpl extends DirectiveUnserialisation {
       priority              <- (directive \ "priority").headOption.flatMap(s => tryo { s.text.toInt } ) ?~! ("Missing or bad attribute 'priority' in entry type directive : " + xml)
       isSystem              <- (directive \ "isSystem").headOption.flatMap(s => tryo { s.text.toBoolean } ) ?~! ("Missing attribute 'isSystem' in entry type directive : " + xml)
       policyMode            =  (directive \ "policyMode").headOption.flatMap(s => PolicyMode.parse(s.text) )
-      directiveIds          =  (directive \ "directiveIds" \ "id" ).map( n => DirectiveId( n.text ) ).toSet
```

Comment:
Yeah, not sur what was the intent of  that code, since directiveIds is never used... Perhaps some debug code that get kept, no idea.